v2.1/demo-follow-the-workload.md

```diff
@@ -165,13 +165,13 @@ The load generator created a `kv` table that maps to an underlying key-value ran
     ~~~
 
     ~~~
-    +----+-----------------+--------+---------------------+---------------------+
-    | id |     address     | build  |     updated_at      |     started_at      |
-    +----+-----------------+--------+---------------------+---------------------+
-    |  1 | localhost:26257 | v1.1.2 | 2017-11-18 05:22:34 | 2017-11-18 05:21:24 |
-    |  2 | localhost:26258 | v1.1.2 | 2017-11-18 05:22:36 | 2017-11-18 05:21:26 |
-    |  3 | localhost:26259 | v1.1.2 | 2017-11-18 05:22:36 | 2017-11-18 05:21:26 |
-    +----+-----------------+--------+---------------------+---------------------+
+    +----+-----------------+--------+---------------------+---------------------+---------+
+    | id |     address     | build  |     updated_at      |     started_at      | is_live |
+    +----+-----------------+--------+---------------------+---------------------+---------+
+    |  1 | localhost:26257 | v2.0.3 | 2018-07-05 07:58:17 | 2018-07-05 07:53:57 | true    |
```

Comment:
As you can see in the `build` column, you're still using v2.0.3. For the v2.1 docs, you need to use the latest v2.1 binary. Can you install that version and update this doc with the appropriate outputs. They're probably the same as in 2.0, but I'd like to be sure.